libsoftwarecontainer/component-test/filegateway_componenttest.cpp

```diff
@@ -69,3 +69,43 @@ TEST_F(FileGatewayTest, TestActivateWithMinimalValidConf) {
     ASSERT_TRUE(gw->setConfig(jsonConfig));
     ASSERT_TRUE(gw->activate());
 }
+
+TEST_F(FileGatewayTest, MalformedPath) {
+// Mount a path which is malformed, e.g. too many "/"
```

Comment:
Yes, agreed. I wrote it in the beginning of the test case to specify what to test, then forgot to redo it as a test header/description.